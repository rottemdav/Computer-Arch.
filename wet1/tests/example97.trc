1 7 24 0 global_history local_tables using_share_mid
0xa1c4 N 0x19f81cc
0x43a34c N 0xe7c33c
0xaeabe2ec T 0x307b8884
0x17ef0c T 0xfbb0
0xad60d98 N 0x16124024
0x5f92dc N 0x99f2dc
0xaeabe2ec N 0x307b8884
0x9f057c N 0xa560
0xae8164c N 0x4d8
0xa1c4 T 0x19f81cc
0x9f057c N 0xa560
0xaeef4 T 0x1acc85c0
0x9f057c T 0xa560
0xad60d98 N 0x16124024
0xfbd516c N 0x52174
0x202c N 0x59d020
0xea3fda74 T 0x518
0xae8164c N 0x4d8
0xea3fda74 N 0x518
0xad60d98 T 0x16124024
0x10e3692c T 0xfe1f80
0x9f40 T 0xd240
0x5f92dc N 0x99f2dc
0xb5940 N 0xc12904
0xae8164c T 0x4d8
0x9f40 T 0xd240
0x5f92dc T 0x99f2dc
0xea3fda74 T 0x518
0x43a34c T 0xe7c33c
0x9f057c N 0xa560
0xbda0 N 0x274
0xfbd516c N 0x52174
0x3d4fb8 T 0xb89c
0x5f92dc N 0x99f2dc
0xbda0 N 0x274
0xad60d98 T 0x16124024
0xaeef4 T 0x1acc85c0
0xae8164c T 0x4d8
0xaba4 T 0xc8577c
0xae8164c N 0x4d8
0x6ef6fc4c T 0x63917884
0x5b6a84 T 0xddc
0x202c T 0x59d020
0x43a34c N 0xe7c33c
0x8801420 N 0x0ea6c
0xbda0 T 0x274
0x9f057c T 0xa560
0xae8164c N 0x4d8
0xbda0 N 0x274
0x43a34c N 0xe7c33c
0xaeabe2ec T 0x307b8884
0x3d4fb8 N 0xb89c
0xae8164c T 0x4d8
0x202c T 0x59d020
0x3d4fb8 T 0xb89c
0xd2bd18 T 0xff959c