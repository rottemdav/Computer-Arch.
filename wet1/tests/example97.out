0xa1c4 N 0xa1c8
0x43a34c N 0x43a350
0xaeabe2ec N 0xaeabe2f0
0x17ef0c N 0x17ef10
0xad60d98 N 0xad60d9c
0x5f92dc N 0x5f92e0
0xaeabe2ec N 0xaeabe2f0
0x9f057c N 0x9f0580
0xae8164c N 0xae81650
0xa1c4 N 0xa1c8
0x9f057c N 0x9f0580
0xaeef4 N 0xaeef8
0x9f057c N 0x9f0580
0xad60d98 N 0xad60d9c
0xfbd516c N 0xfbd5170
0x202c N 0x2030
0xea3fda74 N 0xea3fda78
0xae8164c N 0xae81650
0xea3fda74 N 0xea3fda78
0xad60d98 N 0xad60d9c
0x10e3692c N 0x10e36930
0x9f40 N 0x9f44
0x5f92dc N 0x5f92e0
0xb5940 N 0xb5944
0xae8164c N 0xae81650
0x9f40 N 0x9f44
0x5f92dc N 0x5f92e0
0xea3fda74 N 0xea3fda78
0x43a34c N 0x43a350
0x9f057c N 0x9f0580
0xbda0 N 0xbda4
0xfbd516c N 0xfbd5170
0x3d4fb8 N 0x3d4fbc
0x5f92dc N 0x5f92e0
0xbda0 N 0xbda4
0xad60d98 N 0xad60d9c
0xaeef4 N 0xaeef8
0xae8164c N 0xae81650
0xaba4 N 0xaba8
0xae8164c N 0xae81650
0x6ef6fc4c N 0x6ef6fc50
0x5b6a84 N 0x5b6a88
0x202c N 0x2030
0x43a34c N 0x43a350
0x8801420 N 0x8801424
0xbda0 N 0xbda4
0x9f057c N 0x9f0580
0xae8164c N 0xae81650
0xbda0 N 0xbda4
0x43a34c N 0x43a350
0xaeabe2ec N 0xaeabe2f0
0x3d4fb8 N 0x3d4fbc
0xae8164c N 0xae81650
0x202c N 0x2030
0x3d4fb8 N 0x3d4fbc
0xd2bd18 N 0xd2bd1c
flush_num: 29, br_num: 56, size: 318b
